houston/mission.py

```diff
@@ -20,21 +20,21 @@ class Mission(object):
     configuration = attr.ib(type=Configuration)
     environment = attr.ib(type=Environment)
     initial_state = attr.ib(type=State)
-    commands = attr.ib(type=List[Command])  # FIXME
+    commands = attr.ib(type=Tuple[Command])  # FIXME
```

Comment:
Use a converter to transform a list (or tuple) into a tuple.
See: http://www.attrs.org/en/stable/init.html#converters

Reply:
Nice. I didn't know that exists.